src/client/components/dashboard/CheckerCard.tsx

```diff
@@ -47,16 +48,18 @@ export const CheckerCard: FC<CheckerCardProps> = ({
         onMouseOut={() => setToolbarVisible('hidden')}
         sx={styles.card}
       >
-        <Text sx={styles.title}>{checker.title}</Text>
-        <HStack
-          visibility={isToolbarVisible}
-          mt="50px"
-          mb="35px"
-          justifyContent="center"
-        >
-          <BiDuplicate onClick={onOpen} size="24px" />
-          <BiTrash onClick={onClickDelete} size="24px" />
-        </HStack>
+        <Link to={`/builder/${checker.id}`}>
```

Comment:
@alexanderleegs this is triggering even when clicking on the duplicate/delete buttons..